includes/class-freemius.php

```diff
@@ -17495,13 +17491,17 @@ private function _get_latest_version_endpoint( $addon_id = false, $type = 'json'
          * @param bool|number $addon_id
          * @param bool        $flush      Since 1.1.7.3
          * @param int         $expiration Since 1.2.2.7
+         * @param bool|string $newer_than Since 2.2.1
+         * @param bool|string $readme     Since 2.2.1
          *
          * @return object|false Plugin latest tag info.
          */
         function _fetch_latest_version(
             $addon_id = false,
             $flush = true,
-            $expiration = WP_FS__TIME_24_HOURS_IN_SEC
+            $expiration = WP_FS__TIME_24_HOURS_IN_SEC,
+            $newer_than = false,
+            $readme = true

```

Comment:
@fajardoleo I'd recommend to refactor it to `$include_readme` or `$fetch_readme`.

Reply:
Refactored to `$fetch_readme`.